src/services/edusharing/services/eduSearch.js

```diff
@@ -0,0 +1,19 @@
+//const request = require("request-promise-native");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected exception block, space or tab after '//' in comment.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4791331)